spec/twilio/twilio-reply-spec.js

```diff
@@ -28,7 +28,7 @@ describe('Twilio Reply', () => {
 
   it('sends text messages as a string', done => {
     https.request.pipe(callOptions => {
-      expect(qs.parse(callOptions.body)).toEqual({
+      expect(JSON.parse(JSON.stringify(qs.parse(callOptions.body)))).toEqual({
```

Comment:
if this is to remove extra bits (functions etc) you can just do expect(...).toEqual(jasmine.objectContaining({ /* some props */}))